plugins/Toolbox/resources/qml/ToolboxDetailTile.qml

```diff
@@ -80,8 +80,8 @@ Item
             {
                 background: Rectangle
                 {
-                    implicitWidth: 96
-                    implicitHeight: 30
+                    implicitWidth: 96 * screenScaleFactor
```

Comment:
These should actually use a standardized button class. Obviously that's on me as this is a remnant of my early Plugin Browser tweaks but if we're going to change it; let's change it properly.

Reply:
Do you think we should use an existing button style? Or make a custom one (i.e. a `"toolbox_button"` style) so that it can be more independently modified?

Reply:
How about "toolbox_action_button"?

Reply:
Sure. But you think a separate button style is preferable to using the existing ones? (I'm really asking because I feel constantly on the fence about which UI elements should be able to be independently styled and which ones should be linked to a default style.)

Reply:
Fixed in 89322b9